{{cookiecutter.__project_slug}}/{{cookiecutter.module_name}}/processing.py

```diff
@@ -0,0 +1,19 @@
+from pandas import DataFrame
+
+
+def filter_outliers(data: DataFrame,
+                    column_to_filter: str,
+                    threshold: int) -> DataFrame:
+    '''
+    Filters out all rows from data which have a value larger
+    than the threshold.
+      Parameters:
+        data (DataFrame): Dataframe to filter
```

Comment:
the types (paratheses) of the parameters are already clear from the method parameters.